### DISCUSSION TITLE
Is there a way to change the list of available fonts in the RichEditor form widget?

### DISCUSSION THREAD

**Discussion by nmiyazaki-chapleau** [wintercms/winter#1135] [Q&A] [answered]
FroalaEditor v1.2 (which is what the RichEditor uses) supports custom font family lists through the `fontList` property, but it can only be set on initialization. See [the documentation about fontList](https://froala.com/wysiwyg-editor/v1-2/docs/options/#fontList). Is there any way to change this?
I also tried setting the option manually in the `formwidgets/richeditor/assets/js/richeditor.js` file, but no change I made to that file worked; I ran `php artisan cache:clear`, `php artisan view:clear` and `php artisan winter:util compile assets` to no avail.
Also, is there a way to add fonts to the backend so we can also see the custom fonts working in the richeditor?
Thanks!

**Comment by LukeTowers** [comment, marked as answer]
Perhaps this will help you? https://wintertricks.com/tricks/extend-froala-editor

**Reply by nmiyazaki-chapleau** [reply]
Thanks, that was very helpful! I ended up making a plugin to do this, like in the article. However it turns out I had the wrong version of Froala docs (v1.2 while it was running 2.9.3), which is why fontList never worked... it was fontFamily all along :/ It explains why the hardcoded changes never worked!
